docs/ugcstor.md

```diff
@@ -522,8 +525,113 @@ The deletion of Velero backup schedule doesn't destroy the backup created during
 <h3><a class="anchor" aria-hidden="true" id="Upgrading-the-software-version-of-a-cStor-volume"></a>Upgrading the software version of a cStor volume</h3>
 The steps are mentioned in Upgrade section. For upgrading cStorVolume, ensure that cStor Pool image is support this cStor volume image.  It should also recommended to upgrade the corresponding pool before upgrading cStor volume. The steps for upgrading the cStor volume can be find from [here](/docs/next/upgrade.html).
 
+<h3><a class="anchor" aria-hidden="true" id="Provisioning-sample-application-with-cstor"></a>Provisioning sample application with cstor</h3>
+<br>
+Before provisioning an application ensure that the following steps are completed.
+<ol>
+<li>
+Ensure the disks are mounted as per requirement. 
+To know more about disk mount status <a href="/docs/next/faq.html#what-must-be-the-disk-mount-status-on-node-for-provisioning-openebs-volume" target="_blank">click here</a>.
+</li>
+<li>
+<b>Create StoragePool</b> specifying  the Blockdevices that are to be used.
+To know the detailed steps for creation of StoragePool <a href="/docs/next/ugcstor.html#creating-cStor-storage-pools" target="_blank">click here.</a>
+</li>
+<li>
+Next, you have to <b>create StorageClass</b>, specifying the StoragePoolClaim under annotations.
+<br>To know the step-wise procedure for creation of StoragePool <a href="/docs/next/ugcstor.html#creating-cStor-storage-class" target="_blank">click here</a>.
```

Comment:
Its not creation of StoragePool. It should be `StorageClass`

Reply:
Can you check in Step 3 ... Instead of To know the step-wise procedure for creation of StoragePool -> To know the step-wise procedure for creation of StorageClass

Reply:
This change also reflected in the latest commit.  Only pending is changing disks with blockdevice.

Reply:
I have made all these changes in the last commit. Could you please recheck.